src/components/MenuBar.vue

```diff
@@ -478,6 +469,13 @@ export default defineComponent({
           },
         ];
       }
+      engineMenu.subMenu.push({
+        type: "button",
+        label: "エンジンのフォルダを開く",
```

Comment:
名前何にするかとても迷いますね･･･。
うーん、一旦メインエンジンではないことがわかるよう、「追加エンジンのフォルダを開く」で･･･！
```suggestion
        label: "追加エンジンのフォルダを開く",
```